src/pages/settings/Subscription/CardSection/BillingBanner.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type BillingBannerProps = {
+    title: string;
+    subtitle: string;
+    isError?: boolean;
+    shouldShowRedDotIndicator?: boolean;
+};
+
+function BillingBanner({title, subtitle, isError, shouldShowRedDotIndicator}: BillingBannerProps) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    return (
+        <View style={[styles.pt4, styles.pb3, styles.ph5, styles.flexRow, styles.gap3, styles.w100, styles.alignItemsCenter, styles.hoveredComponentBG]}>
+            <Icon
```

Comment:
@Expensify/design Would we have a case where we might not want to show any icon?

Reply:
Let's confirm with @dannymcclain for this one too.

Reply:
We currently do not have any cases where we don't have an icon. And playing around with it in Figma, the icon really contributes a lot to the layout so I'm inclined to say we should always include one.